compiler/src/iree/compiler/API/Internal/Embed.cpp

```diff
@@ -543,6 +542,13 @@ Invocation::Invocation(Session &session)
     mlir::applyDefaultTimingPassManagerCLOptions(passManager);
   }
   passManager.addInstrumentation(std::make_unique<PassTracing>());
+
+  pipelineHooks.buildConstEvalPassPipelineCallback = [](OpPassManager &pm) {
```

Comment:
I missed this last time, could we add comment why this pass is treated specially here?